src/status_im/navigation/options.cljs

```diff
@@ -100,7 +100,7 @@
   {:layout                 {:componentBackgroundColor :transparent
                             :orientation              ["portrait"]
                             :backgroundColor          :transparent}
-   :modalPresentationStyle :overCurrentContext
+   :modalPresentationStyle :overFullScreen
```

Comment:
Hi @ibrkhalil,
Thank you very much for looking into this issue.

`modalPresentationStyle` will change how modal screens are presented. As [overFullScreen](https://wix.github.io/react-native-navigation/api/options-root#modalpresentationstyle) is iOS-specific we also need to make sure how it looks in Android.

To me, change seems ok, but I think @OmarBasem  will be the right person to ask this question because he [introduced](https://github.com/status-im/status-mobile/pull/15399) this screen and knows what is the expected behavior here.

Also, If we don't want to use bottom-sheet-screen component for these screens then we can keep these changes, otherwise I have [fixed](https://github.com/status-im/status-mobile/pull/18886#pullrequestreview-1887331452) the underlying issues with multiple screens navigation problem.

Reply:
@Parveshdhull @ibrkhalil using `overFullScreen` causes the bottom sheet screen to not look right on Android. The background opacity layer is not there anymore. And while scrolling the bottom sheet up and down there is just a black background

https://github.com/status-im/status-mobile/assets/29354102/a0b1447c-5534-451f-82a7-f3c19c361f6b